mailing_list/models.py

```diff
@@ -151,8 +185,7 @@ def address(self):
             section_id=self.section_id
         )
 
-    @property
-    def access_level(self):
+    def get_listserv_access_level(self):
```

Comment:
I had to remove it as a property when it was  called 'access_level' - as I introduced a new field with the same name. Since it is renamed, can make it a property again